src/main/java/mate/jdbc/serviceimpl/DriverServiceImpl.java

```diff
@@ -0,0 +1,44 @@
+package mate.jdbc.serviceimpl;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+import mate.jdbc.dao.DriverDao;
+import mate.jdbc.dao.DriverDaoImpl;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Driver;
+import mate.jdbc.service.DriverService;
+
+@Service
+public class DriverServiceImpl implements DriverService {
+    @Inject
+    private DriverDao driverDao = new DriverDaoImpl();
+
+    @Override
+    public Driver create(Driver driver) {
+        return driverDao.create(driver);
+    }
+
+    @Override
+    public Driver get(Long id) {
+        if (driverDao.get(id).isEmpty()) {
+            throw new NoSuchElementException("Can't get manufacturer with ID " + id);
+        }
+        return driverDao.get(id).get();

```

Comment:
```suggestion
        return driverDao.get(id).orElseThrow();

```
throw error explicitly where possible